rep-2000.rst

```diff
@@ -172,6 +172,112 @@ Middleware Implementation Support:
 - RTI Connext
 - ADLink OpenSplice
 
+Crystal Clemmys (December 2018 - TBD, at least December 2019)
+-------------------------------------
+
+Support Tiers:
+
+Tier 1
+
+Tier 1 platforms are subjected to our unit test suite and other testing tools on a frequent basis including continuous integration jobs, nightly jobs, packaging jobs, and performance testing.
+Errors or bugs discovered in these platforms are prioritized for correction by the development team.
+Significant errors discovered in Tier 1 platforms can impact release dates and we strive to resolve all known high priority errors in Tier 1 platforms prior to new version releases.
+
+Tier 2
+
+Tier 2 platforms are subject to ad hoc testing and binary packages or archives may not be provided or supported.
```

Comment:
Should we consider to offer at least archives for Tier 2 platforms? That would more clearly distinguish them from Tier 3.